SpinCore_pp/ppg/generic.py

```diff
@@ -32,6 +31,7 @@ def generic(
     ph2_cyc=r_[0],
     ret_data=None,
     amplitude=1.0,
+    manual_echoes=0,
```

Comment:
you need to merge master into this branch to get rid of extra  diffs  now

Reply:
(on a PR, if you update master, the website won't update until you merge master into the branch)